views/job-roles.html

```diff
@@ -0,0 +1,30 @@
+{% extends "_includes/layouts/base.html" %}
```

Comment:
There is a bug on this page in that some of the data is cut off because of the footer. If you try and scroll nothing happens